.gitignore

```diff
@@ -5,3 +5,5 @@ dist/
 **/RCS/**
 .idea
 .ropeproject
+data/triage-tools-tickets
+.vscode
```

Comment:
No need to change it as there are more of the same, but normally personal editor/settings files to be ignored it's best to set them in global gitignore